garak/cli.py

```diff
@@ -162,7 +162,7 @@ def main(arguments=None) -> None:
         "-p",
         type=str,
         default=_config.plugins.probe_spec,
-        help="list of probe names to use, or 'all' for all (default).",
+        help="list of probe names to use, or 'all' for all (default). For filtering with --list_probes, use e.g. '--list_probes -p dan' to show only 'dan' family probes.",
```

Comment:
Please revert this line, the combination is only in effect if `--list_probes` or `--list_detectors` is used, the comment about combination should only be on those options.